code/modules/reagents/chemistry_properties/prop_positive.dm

```diff
@@ -196,20 +196,21 @@
 /datum/chem_property/positive/painkilling/process(mob/living/M, var/potency = 1, delta_time)
 	if(!..())
 		return
-
-	M.pain.apply_pain_reduction(PAIN_REDUCTION_MULTIPLIER * potency)
+	var/effective_potency = (CHECK_BITFIELD(M.disabilities, OPIATE_RECEPTOR_DEFICIENCY) ? potency * 0.25 : potency)
```

Comment:
probably OOS but this would probably be best implemented as a mob/living variable that controls painkiller potency rather than checking for a character quirk for future Usage